src/main/webapp/content/scss/global.scss

```diff
@@ -802,41 +802,9 @@ Course Info Bar
         display: inline;
     }
 
-    ol,
-    ul {
-        margin: 0;
-        padding: 0;
-        list-style: none;
-    }
-
-    ol {
-        counter-reset: item;
-    }
-
-    ol > li,
-    ul > li {
-        position: relative;
-        padding-left: 1em;
-        margin-bottom: 0.5em;
-        line-height: 1;
-        text-indent: 0;
-    }
-
-    ol > li::before {
-        content: counter(item) '.';
-        counter-increment: item;
-        position: absolute;
-        left: 0;
-        top: 0;
-        font-weight: normal;
-    }
-
-    ul > li::before {
-        content: '•';
-        position: absolute;
-        left: 0;
-        top: 0;
-        font-weight: bold;
+    ol > li:last-child,
+    ul > li:last-child {
+        margin-bottom: 0.75em;
```

Comment:
You could add a comment and a markdown example here for what we need it, otherwise it will be quite unclear which case this shall cover, in case this breaks something in the future